gwsumm/archive.py

```diff
@@ -456,7 +460,7 @@ def archive_table(table, key, parent):
     except KeyError:
         pass
     for col in table.columns:
-        if table[col].dtype.type is unicode_:
+        if table[col].dtype.type is str_:
```

Comment:
@eagoetz, `numpy.str_` (and `numpy.unicode_`) both repond positively to `isinstance(..., str)` and similar, so you might be able to simplify things with

```suggestion
        if numpy.issubdtype(table[col].dtype, str):
```

without having to import anything from `numpy` conditional on version.